pkg/server/virtual.go

```diff
@@ -115,6 +115,8 @@ func (s *Server) installVirtualWorkspaces(
 		return err
 	}
 
+	preparedRootAPIServer.AuditBackend = server.AuditBackend
```

Comment:
I think I added this back before we had audit working correctly, but IIRC @s-urbaniak got it wired in correctly, so we can probably remove this (?). It looks like line 92 has it covered?

Reply:
Ah yes thanks, removed